nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/FXMLController.java

```diff
@@ -863,6 +861,39 @@ public void initialize(URL url, ResourceBundle rb) {
         borderPane.heightProperty().addListener(e -> contentController.updateScrollSize(borderPane));
     }
 
+    /**
+     * Initialize the toggle buttons Phasing, Attributes and Contents. On mac these buttons will appear right
+     * aligned in a separate top menu in the window, otherwise they will appear right aligned in the file menu.
+     */
+    public void initializeToggleButtons() {
```

Comment:
method should be private and maybe have a more explicit name? There could be other toggle buttons somewhere in the UI.

Reply:
changed to private, not really sure what to call these buttons but changed method name to `initializeRightPaneContentControlToggleButtons`